server/indexers/transactions.ts

```diff
@@ -12,25 +17,42 @@ export default async function transactions(connection: PoolClient, provider: eth
   async function process() {
     const bids = await findBidsWithMissingTransactions.run({ limit: 5 }, connection);
 
-    log.debug(`Rows: ${bids.length}`);
-
     if (bids.length === 0) {
+      const latestBids = await findWalletsInLatestAuction.run(undefined, connection);
+
+      log.debug(`Rows: ${bids.length} ${latestBids.length}`);
+
+      const latestBalances = await Promise.all(
+        latestBids.map((row) => provider.getBalance(row.walletAddress)),
+      );
+
+      for (const [index, row] of latestBids.entries()) {
+        await updateWalletsValuesInLatestAuction.run(
+          {
+            tx: row.tx,
+            index: row.index,
+            walletBalance: latestBalances[index].toString() || null,
```

Comment:
what do you  need `|| null` for? `latestBalances[index].toString()` will never be falsy.

The code below is `balancesInBlocks[index]?.toString()` and tries to defend against `balancesInBlocks[index]?` being undefined, but I struggle to see how it could be the case